x-pack/plugins/apm/public/components/shared/time_comparison/get_comparison_options.ts

```diff
@@ -120,10 +143,35 @@ export function getComparisonOptions({
     comparisonTypes = [TimeRangeComparisonEnum.PeriodBefore];
   }
 
-  return getSelectOptions({
+  const hasMLJobsMatchingEnv =
+    canGetJobs &&
+    Array.isArray(anomalyDetectionJobsData?.jobs) &&
+    anomalyDetectionJobsData?.jobs.some(
+      (j) => j.environment === preferredEnvironment
+    );
+
+  const comparisonOptions = getSelectOptions({
     comparisonTypes,
     start: momentStart,
     end: momentEnd,
     msDiff,
   });
+
+  if (canGetJobs) {
+    const disabled =
+      anomalyDetectionJobsStatus === 'success' && !hasMLJobsMatchingEnv;
+    comparisonOptions.push({
+      value: TimeRangeComparisonEnum.ExpectedBounds,
+      text: disabled
+        ? i18n.translate('xpack.apm.comparison.mlExpectedBoundsDisabledText', {
+            defaultMessage: 'Expected bounds (select other environment)',
```

Comment:
We should mention that anomaly detection must be enabled for the environment:

```suggestion
            defaultMessage: 'Expected bounds (anomaly detection must be enabled for env)',
```

Reply:
@sqren Thanks. Updated with the new text here... I capitalized Anomaly so it looks more like this now: https://github.com/elastic/kibana/pull/132456/commits/1d889d0393896c39d68de060e837dc95ccfe9652

<img width="621" alt="Screen Shot 2022-05-24 at 16 25 59" src="https://user-images.githubusercontent.com/43350163/170136929-5352ec27-901e-4d26-af45-f7f426d885fc.png">